doc/devdocs/readme.md

```diff
@@ -80,9 +80,24 @@ modify --installpath "%ProgramFiles(x86)%\Microsoft Visual Studio\2019\%targetFo
 - The PowerToys binaries will be in your repo under `x64\Release`.
 - If you want to copy the `PowerToys.exe` binary to a different location, you'll also need to copy the `modules` and the `svgs` folders.
 
+### Temporary build-fix for Toolset v142 (Visual Studio 2019 Version 16.8)
+
+When you use the Toolset v142 (came  typical with Visual Studio 16.8) and you cannot build the C++ project `common-md-flag` you can try one of the follow solutions.
+
+Add the two compiler switches `/await` and `/Zc:twoPhase-` to the `common-md-flag` project.
+
+![Project property change](/doc/images/v142PropertyChange.png)
+
+or add the compiler switch `/await` to the `common-md-flag` project and change the Windows SDK version of the project to version `10.1.18362.0`.
+
+Note: You must install the [Windows SDK 10.1.18362.0](https://developer.microsoft.com/en-us/windows/downloads/sdk-archive/) before you can switch to it.
```

Comment:
Why is this a new requirement?  It is unclear

Reply:
1. The PowerShell script don't install the `Windows SDK 10.1.18362.0`.
2. When you haven't install the SDK with the version you can't select it for a C++ project.

Reply:
> The PowerShell script don't install the Windows SDK 10.1.18362.0.

The SDK to use is 10.0.17134.0 not 10.1.18362.0.